src/muqsit/worldstyler/schematics/Schematic.php

```diff
@@ -28,7 +29,7 @@ public function __construct(string $file)
 
     public function load() : void
     {
-        $this->namedtag = (new BigEndianNBTStream())->readCompressed(file_get_contents($this->file))->getTag();
+        $this->namedtag = (new BigEndianNbtSerializer())->read(file_get_contents($this->file))->getTag();
```

Comment:
file contents need decompressed before they can be read

Reply:
That^. `BigEndianNbtSerializer::read` and `BigEndianNbtStream::readCompressed` aren't the same. You must
```php
$this->namedtag = (new BigEndianNbtSerializer())->read(zlib_decode(file_get_contents($this->file)))->getTag();
```